io_scene_nif/modules/nif_export/property/texture/types/bsshadertexture.py

```diff
@@ -85,28 +85,44 @@ def export_bs_lighting_shader_prop_textures(self, bsshader):
         if self.slots["Gloss"]:
             texset.textures[7] = TextureWriter.export_texture_filename(self.slots["Gloss"])
 
-        # TODO [shader] UV offset node support
-        # # UV Offset
-        # if hasattr(bsshader, 'uv_offset'):
-        #     self.export_uv_offset(bsshader)
-        #
-        # # UV Scale
-        # if hasattr(bsshader, 'uv_scale'):
-        #     self.export_uv_scale(bsshader)
+        #get the offset, scale and UV wrapping mode and set them
+        x_scale, y_scale, x_offset, y_offset, clamp_x, clamp_y = self.get_global_uv_transform_clip(self.slots["Base"])
```

Comment:
"Base" can be pulled out to constants

Reply:
What do you mean by this?

Reply:
For any of names that we use specifically for lookup, we should pull them out as constants, either within the class or the global const classes.